addons/web/static/src/js/core/utils.js

```diff
@@ -267,6 +267,21 @@ var utils = {
     confine: function (val, min, max) {
         return Math.max(min, Math.min(max, val));
     },
+    /**
+     * Looks through the list and returns the first value that matches all
+     * of the key-value pairs listed in properties.
+     * If no match is found, or if list is empty, undefined will be returned.
+     *
+     * @param {Array} list
+     * @param {Object} props
+     * @returns 
+     */
+    findWhere: function (list, props) {
+        if (!Array.isArray(list)) {
+            return;
+        }
+        return list.find(item => Object.keys(props).every(key => item[key] === props[key]));
+    },
```

Comment:
for the commit message, I would go for, `[IMP] web: add findWhere utils`, ADD being rather for new modules or big features.
Also, shouldn't you test it? :D